modules/varautoencoder_mednist.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>it seems <code>loss_balance = []</code>  is unused and could be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>